packages/react/src/FormattingToolbar/components/DefaultButtons/ToggledStyleButton.tsx

```diff
@@ -9,30 +12,35 @@ import {
   RiUnderline,
 } from "react-icons/ri";
 
+import { StyleSchema } from "@blocknote/core";
 import { ToolbarButton } from "../../../SharedComponents/Toolbar/components/ToolbarButton";
 import { useEditorChange } from "../../../hooks/useEditorChange";
 import { useSelectedBlocks } from "../../../hooks/useSelectedBlocks";
 import { formatKeyboardShortcut } from "../../../utils";
 
-const shortcuts: Record<ToggledStyle, string> = {
+const shortcuts = {
   bold: "Mod+B",
   italic: "Mod+I",
   underline: "Mod+U",
   strike: "Mod+Shift+X",
   code: "",
 };
 
-const icons: Record<ToggledStyle, IconType> = {
+const icons = {
   bold: RiBold,
   italic: RiItalic,
   underline: RiUnderline,
   strike: RiStrikethrough,
   code: RiCodeFill,
 };
 
-export const ToggledStyleButton = <BSchema extends BlockSchema>(props: {
-  editor: BlockNoteEditor<BSchema>;
-  toggledStyle: ToggledStyle;
+export const ToggledStyleButton = <
```

Comment:
We probably want to make this extensible, so you can easily make buttons for custom styles.

Reply:
Yes, can you give this a go? I think we mainly want to move `shortcuts` and `icons` to DefaultToolbar